deploy/054_deploy_alephzero_spokepool.ts

```diff
@@ -0,0 +1,32 @@
+import { DeployFunction } from "hardhat-deploy/types";
+import { HardhatRuntimeEnvironment } from "hardhat/types";
+import { deployNewProxy, getSpokePoolDeploymentInfo } from "../utils/utils.hre";
+import { FILL_DEADLINE_BUFFER, L2_ADDRESS_MAP, QUOTE_TIME_BUFFER, WAZERO, ZERO_ADDRESS } from "./consts";
+
+const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
+  const { hubPool, spokeChainId } = await getSpokePoolDeploymentInfo(hre);
+
+  const initArgs = [
+    // Initialize deposit counter to very high number of deposits to avoid duplicate deposit ID's
+    // with deprecated spoke pool.
+    0,
```

Comment:
The comment here is inconsistent w/ the initial deposit ID. I'd probably just nuke the comment.

```suggestion
    0,
```